app/User.php

```diff
@@ -27,8 +27,14 @@ class User extends Authenticatable
         'password', 'remember_token',
     ];
 
+
     public function notes()
     {
         return $this->hasMany('App\Note');
     }
+
+    public function centre()
```

Comment:
Boring of me to say - but missing doc comments.

Reply:
okedoke.